tests/cases/fourslash/convertFunctionToEs6Class_commentOnVariableDeclaration.ts

```diff
@@ -8,7 +8,8 @@
 verify.codeFix({
     description: "Convert function to an ES2015 class",
     newFileContent:
-`/** Doc */
+`
```

Comment:
This doesn't seem great.  What is the comment being inserted after that is resulting in the blank line?

Reply:
The way `convertFunctionToEs6Class` works, we are deleting the function and adding a class node after it. In textChanges we don't know that the function is going to be deleted, so we add a newline to separate the old function and the new class. Added an issue at #23871